### DISCUSSION TITLE
Can it work with dynamic attributes?

### DISCUSSION THREAD

**Discussion by brybrant** [ivodolenc/nuxt-lazysizes#16] [Q&A]
### Steps to Reproduce

Hi. I have images with dynamic sources like this:

```vue
<template>
  <img
    class="lazyload"
    :src="`~/assets/${month}--low.jpg`"
    :data-src="`~/assets/${month}.jpg`"
  >
</template>

<script>
export default {
  data () {
    return {
      month: ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec'][new Date().getMonth()]
    }
  }
}
</script>
```

Can nuxt-lazysizes be made to work with dynamic attributes like this?

### Current Behavior

Attribute is not processed

### Expected Behavior

Process dynamic attributes

### Version

1.4.3

**Comment by ivodolenc** [comment]
Hi, have you tried [`extendAssetUrls`](https://github.com/ivodolenc/nuxt-lazysizes#advanced-usage-optional) option?

```ts
// nuxt.config.js

export default {
  lazySizes: {
    extendAssetUrls: {
      img: ['src', 'data-src'],
    }
  }
}
```

**Reply by brybrant** [reply]
Yep, I tried that. It works when I set the attribute normally (not dynamic) like:
```vue
<img class="lazyload" data-src="~/assets/example.png">
```

If I want to use dynamic attribute (with colon) then it only works if I use `require()` like:
```vue
<img class="lazyload" :data-src="require('~/assets/example.png')">
```

I will just move my images to the `/static` folder instead of the `/assets` folder to make things simple.
